app/controllers/admin/student_accounts_controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+module Admin
+    class StudentAccountsController < BaseController
+      layout 'admin'
+  
+      def actions
+      end
+  
+      def get_number_of_accounts
+        days_num = get_number_of_days
+        accounts_num = query(days_num)
```

Comment:
I would rename `accounts_num` to `result`. But also note that after removing `get_number_of_days`, these two lines will just be `result = query` which is okay I suppose (although there might be no need to do that and just call `query` where needed.

Reply:
I renamed it to be more descriptive.